src/main/java/pb360/service/GenerateRestApiService.java

```diff
@@ -0,0 +1,18 @@
+package pb360.service;
+
+import pb360.model.MessageObject;
+
+public interface GenerateRestApiService {
+	public MessageObject getRestApiData(String id);
+
+	public MessageObject searchRestApiData();
+
+	public MessageObject postRestApiData();
+
+	public MessageObject patchRestApiData();
+
+	public MessageObject deleteRestApiData();
+
+	public MessageObject getCurrentDateTime();
```

Comment:
remove this function

Reply:
+	public MessageObject getCurrentDateTime();  removed 
but should i remove deleteRestApiData()?